renderer/components/Gamepad.tsx

```diff
@@ -8,8 +8,13 @@ import {
 import _ from 'lodash';
 import numeral from 'numeral';
 
+export interface GamepadObject{
+  axes: Array<String | number> | object;
+  buttons: Array<String | number> | object;
+}
+
```

Comment:
Note: I specifically made a separate interface for `gamepad` in `OwnProps` because it needs to be reused in `GamepadList.tsx`.

Reply:
Ok I see, in that case we can move this to the `types` folder.